build.go

```diff
@@ -63,13 +60,15 @@ func (r *Runner) InstallGems(packages []string) error {
 	return nil
 }
 
-func (r *Runner) CheckPackageExists(packages []string) []string {
-	for _, p := range packages {
-		path, err := exec.LookPath(p)
-		if err != nil {
-			fmt.Printf("didn't find '%s' package\n", p)
-		}
-		fmt.Printf("'%s' package is '%s'\n", p, path)
+func (r *Runner) IsInstalled(pkg string) bool {
+	err := r.Command("rpm -q", pkg)
+
+	var exiterror *exec.ExitError
```

Comment:
Yes, this works, but thinking about it a little more, we don't actually _need_ the exit status, or even to know whether the error was an `os.ExitError`, do we? We _know_ that on a properly configured CentOS system the `rpm -q foo` command should work, so if it errors, the only plausible reason for that is that the package is not installed.

So in fact we could simply write here:

```suggestion
	return err == nil
```